rust/README.md

```diff
@@ -51,8 +59,24 @@ cargo run --example read_csv
 
 ## Run Tests
 
+Parquet support in Arrow requires data to test against, this data is in a
+git submodule.  To pull down this data run the following:
+
+```bash
+git submodule update --init
+```
+
+The data can then be found in `cpp/submodules/parquet_testing/data`.
+Create a new environment variable called `PARQUET_TEST_DATA` to point
```

Comment:
We should set this as the default if `PARQUET_TEST_DATA` is not defined.

Reply:
Yep, also a good idea.  Thanks @kszucs